__tests__/app.tests.js

```diff
@@ -68,13 +67,62 @@ describe("GET api/articles/ID ", () => {
         expect(output).toHaveProperty("article_img_url");
       });
   });
+  test("GET articles by id should return article with properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        const output = res.body.articles[0];
+        expect(output.author).toEqual(expect.any(String));
+        expect(output.title).toEqual(expect.any(String));
+        expect(output.body).toEqual(expect.any(String));
+        expect(output.topic).toEqual(expect.any(String));
+        expect(output.created_at).toEqual(expect.any(String));
+        expect(output.article_img_url).toEqual(expect.any(String));
+        expect(output.article_id).toEqual(expect.any(Number));
+        expect(output.votes).toEqual(expect.any(Number));
+      });
+  });
   test("GET articles by id should return a 404 and msg of 'no such article !' if id is not in the db", () => {
     return request(app)
       .get("/api/articles/90")
       .expect(404)
       .then((res) => {
-        console.log(res.body.msg);
         expect(res.body.msg).toEqual("No such article !");
       });
   });
 });
+describe("GET api/article", () => {
+  test("GET API/article should return an array of article objects ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const output = res.body.articles[0];
+        expect(typeof res.body).toEqual("object");
+        expect(output).toHaveProperty("author");
+        expect(output).toHaveProperty("title");
+        expect(output).toHaveProperty("article_id");
+        expect(output).toHaveProperty("topic");
+        expect(output).toHaveProperty("created_at");
+        expect(output).toHaveProperty("votes");
+        expect(output).toHaveProperty("article_img_url");
+      });
+  });
+  test("GET articles with correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const output = res.body.articles[0];
+        expect(output.author).toEqual(expect.any(String));
+        expect(output.title).toEqual(expect.any(String));
+        expect(output.body).toEqual(expect.any(String));
+        expect(output.topic).toEqual(expect.any(String));
+        expect(output.created_at).toEqual(expect.any(String));
+        expect(output.article_img_url).toEqual(expect.any(String));
+        expect(output.article_id).toEqual(expect.any(Number));
+        expect(output.votes).toEqual(expect.any(Number));
+      });
+  });
```

Comment:
These are good tests Guy well done, they allow for expansion later down the line. 

I think they should be wrapped within a forEach, currently you are only asserting that the first article has these properties, but that doesn't prove anything for the subsequent articles 👍

Reply:
You are also missing a comment count property on these tests

Reply:
You should include tests that assert the sort order as well, Ie created_at, desc. You can use jest-sorted package for this. 

https://www.npmjs.com/package/jest-sorted